grml-debootstrap

```diff
@@ -979,7 +979,9 @@ efi_support() {
     einfo "EFI support detected, but system seems to be running in BIOS mode."
   fi
 
-  return 1
+  # The user may have a legitimate reason to do an EFI installation on a BIOS
+  # system (for instance if doing a hybrid-bootable installation), so don't
+  # return 1 here.
```

Comment:
if this function never returns 1, the !efi-support code paths can go away

Reply:
Good point. See also my reply to mika at https://github.com/grml/grml-debootstrap/pull/299#discussion_r1968681140.

Reply:
Removed the unneeded code path.